account_asset_management/tests/test_account_asset_management.py

```diff
@@ -640,18 +640,23 @@ def test_15_account_asset_group(self):
         )
         # Groups are displayed by code (if any) plus name
         self.assertEqual(
-            self.env["account.asset.group"]._name_search("FA"),
+            self.env["account.asset.group"]
+            .browse(self.env["account.asset.group"]._name_search("FA"))
+            .name_get(),
```

Comment:
Although I would change the second operand to match instead of calling another method, I think it's better to test public APIs, as they are the ones that will be called in UI/reality. Can you call `name_search` instead?